templates/pages/admin/form/form_destination.html.twig

```diff
@@ -97,11 +96,21 @@
                                 {# Submit button to delete the destination #}
                                 {% if can_update %}
                                     <div class="d-flex flex-row-reverse mt-3">
-                                        <button class="btn btn-primary ms-2" name="update" type="submit">
+                                        <button
+                                            class="btn btn-primary ms-2"
+                                            name="update"
+                                            type="submit"
+                                            formaction="{{ '/Form/%s/Destination/%s/Update'|format(form.getID(), destination.getID()) }}"
```

Comment:
Not a big deal, but it could permit to be sure to trigger an error if IDs are not integers.
```suggestion
                                            formaction="{{ '/Form/%d/Destination/%d/Update'|format(form.getID(), destination.getID()) }}"
```